src/routes/api/users/UserController/UserController.js

```diff
@@ -0,0 +1,25 @@
+import db from '../../../../models/index';
+/**
+ * @class Users
+ */
+class Users {
+  /**
+   * @static
+   * @param {obj} req
+   * @param {obj} res
+   * @returns {obj} res
+   * @memberof Users
+   */
+  static async getAllUsers(req, res) {
+    const users = await db.user
+      .findAll({
+        attributes: ['username', 'firstname', 'lastname', 'email', 'updatedAt', 'createdAt', 'bio', 'image']
```

Comment:
Change attributes to exclude

Reply:
Thanks for the feedback. I have implemented the feedback.